revolv/payments/services.py

```diff
@@ -13,12 +15,14 @@ class PaymentServiceException(Exception):
 class PaymentService(object):
 
     @classmethod
-    def create_payment(cls, user, amount, payment_instrument):
+    def create_payment(cls, user, entrant, amount, project, payment_instrument):
```

Comment:
I don't think so - if we changed this method to accept a namedtuple instead, then we would just have to instantiate a namedtuple with all the arguments somewhere else